bitshuffle/bitshuffle.py

```diff
@@ -179,7 +182,6 @@ def main():
 
     parser.add_argument("--editor", "-E",
                         help="Editor to use for pasting packets. " +
-                        "If not specified, defaults in this order:\n" +
```

Comment:
What's the thought behind removing this?

Reply:
I had not intended to remove this... whoops!